data/src/main/java/com/example/data/database/entities/ListDB.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.data.database.entities
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.example.data.Utils.ConstantUtils
+
+@Entity
```

Comment:
This could not be @Entity since it's not stored in the database directly but as a String.
Same for it's inner classes

Reply:
Done